src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -223,22 +223,26 @@ int64_t ClusterResourceScheduler::GetBestSchedulableNode(const TaskRequest &task
   // We will pick the node with the smallest number of soft violations.
   int64_t min_violations = INT_MAX;
   // Node associated to min_violations.
-  int64_t best_node = -1;
+  std::vector<int64_t> best_nodes;
   *total_violations = 0;
 
   if (actor_creation && task_req.IsEmpty()) {
```

Comment:
Why does the 0 cpu case need to be modified at all? This change looks like it will results in the same behavior.

Reply:
Not a big deal, just wanted to make sure I understood correctly. (though now it looks like we could just return immediately instead of breaking...).